python/paddle/fluid/transpiler/distribute_transpiler.py

```diff
@@ -328,8 +333,93 @@ def get_trainer_program(self):
         # FIXME(typhoonzero): Also ops like clip_gradient, lrn_decay?
         delete_ops(self.origin_program.global_block(), self.optimize_ops)
         self.origin_program.__str__()
+
         return self.origin_program
 
+    def _get_trainer_startup_program(self, program=None):
+        """
+        Get transpiled trainer side startup program.
+
+        Returns:
+            Program: trainer side startup program.
+        """
+        if program is None:
+            program = self.startup_program
+
+        orig_s_prog = program
+
+        #delete initialize operators.
+        assert (orig_s_prog.num_blocks == 1)
```

Comment:
Why need this assert, we currently have no places will have multiple blocks in the startup program

Reply:
Done.